src/mysql.c

```diff
@@ -433,121 +436,204 @@ static int mysql_read_primary_stats(mysql_database_t *db, MYSQL *con) {
 
 static int mysql_read_replica_stats(mysql_database_t *db, MYSQL *con) {
   MYSQL_RES *res;
+  MYSQL_FIELD *fields;
   MYSQL_ROW row;
 
-  const char *query;
-  int field_num;
+  const char *query, *channel_name_field;
+  unsigned char field_num, field_req;
```

Comment:
Why 8-bit variable instead of e.g. (unsigned) int?

Reply:
reduce usage ram, unsigned char - 1 byte, AFAIK

Reply:
Performance claims should be backed up by evidence.  :-)

What e.g. "file collecd" states for compiled collectd code (text section) sizes when you use unsigned int vs. unsigned char?